moveit_ros/planning_interface/planning_scene_interface/src/planning_scene_interface.cpp

```diff
@@ -48,8 +48,9 @@ namespace planning_interface
 class PlanningSceneInterface::PlanningSceneInterfaceImpl
 {
 public:
-  PlanningSceneInterfaceImpl()
+  explicit PlanningSceneInterfaceImpl(const std::string& ns = "")
   {
+    node_handle_ = ros::NodeHandle(ns);
```

Comment:
This local variable shouldn't have a trailing underscore.

Reply:
Correct, I think this was a rebase artifact as I had previously stored this namespace as a private variable in the class for some functionality that did was not targeted for this PR